README.md

```diff
@@ -42,7 +42,9 @@ NOTE: Requires k3d<br><br>
 
 Tilt takes care of creating resouces and giving you access to the logs, as well as creating any port-forwarding you need. You'll have easy access from tilt's builtin dashboard.<br>
 
-To spin up a k3d observability cluster, and deploy the default LGTM stack with tilt; run `make tilt-up`. <br>
+To spin up a k3d otel-basic cluster, and deploy the default LGTM stack with tilt; run `make tilt-basic`. <br>
```

Comment:
This is more of a note for myself, we'll probably want to change the `basic` term to something else. Doesn't need to be done here though. Will be a future iteration.